blueocean-core-js/src/js/paths/rest.js

```diff
@@ -23,7 +27,7 @@ export default {
 
     run({ organization, pipeline, branch, runId }) {
         if (branch) {
-            return `${this.pipeline(organization, pipeline)}branches/${branch}/runs/${runId}/`;
+            return `${this.pipeline(organization, pipeline)}branches/${this.encodeSlashes(branch)}/runs/${runId}/`;
```

Comment:
-1 - this wasn't used before and a different method was used. It should encode things the way it did previously (I think there was a more general encoding funciton)>